addons/portal/controllers/mail.py

```diff
@@ -193,9 +214,16 @@ def portal_message_fetch(self, res_model, res_id, domain=False, limit=10, offset
             if not request.env['res.users'].has_group('base.group_user'):
                 domain = expression.AND([Message._get_search_domain_share(), domain])
             Message = request.env['mail.message'].sudo()
+
+        message_ids = Message.search(domain, limit=limit, offset=offset, order='date desc')
+        messages = message_ids.portal_message_format()
+        domain_next_attachment_ids = ['&', ('attachment_ids', '!=', False), ('id', 'not in', message_ids.ids)]
+        message_next_attachments_ids = Message.search(expression.AND([domain_next_attachment_ids, domain]))
+
         return {
-            'messages': Message.search(domain, limit=limit, offset=offset).portal_message_format(),
-            'message_count': Message.search_count(domain)
+            'messages': messages, # 'grouped_messages': self._prepare_portal_message_fetch_group_by_day(messages, message_ids),
```

Comment:
same here ? the comment is a reminiscence of the past (like my mother used to say) ?